<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabConnect</class>
 <widget class="QMainWindow" name="LabConnect">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>408</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>LabConnect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>100000.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>11</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frequenz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>2</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>4</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>1 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 MHz</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>10</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Manuell setzen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MHz</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>326</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ausgabe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>177</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Signalform</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Sinus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dreieck</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rechteck</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>213</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ausgangsspannung</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Offsetspannung</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>256</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLabConnect">
    <property name="title">
     <string>LabConnect</string>
    </property>
    <addaction name="actionGer_te_neu_laden"/>
    <addaction name="actionEinstellungen_2"/>
    <addaction name="separator"/>
    <addaction name="actionAlle_Verbindungen_trennen"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menu_ber">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionCredits"/>
    <addaction name="actionVersion"/>
    <addaction name="separator"/>
    <addaction name="actionHilfe"/>
   </widget>
   <addaction name="menuLabConnect"/>
   <addaction name="menu_ber"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGer_te_neu_laden">
   <property name="text">
    <string>Ger√§te neu laden</string>
   </property>
  </action>
  <action name="actionAlle_Verbindungen_trennen">
   <property name="text">
    <string>Alle Verbindungen trennen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionEinstellungen_2">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
